% !TEX root = ../main.tex
\appendix
\chapter{Appendix}

\section{HPLC Methods} % (fold)
\label{sec:hplc_methods}

	Vielleicht wenn hier text steht

	\begin{table}[htbp]
		\caption[Standard C18 screening method]{\textbf{Standard C18 screening method}}
		\label{tab:method_c18_screening}
		\centering
		\begin{tabularx}{\textwidth}{XX}
			\toprule
			\textbf{Parameter}	& \textbf{Value}	\\
			\midrule
			Column 		& Nucleosil-100 C18 \SI{5}{\micro\meter} 150$\times$\SI{3}{\milli\meter} 	\\
			Solvents	& A: Water + 0.1~\% Formic acid 	\\
						& B: Acetonitrile + 0.1~\% Formic acid		\\
			Method 		& Gradient 5 - 100 \% B for \SI{15}{\minute} 	\\
						& Plateau 100 \% B for \SI{3}{\minute} 	\\
			Flow 		& \SI{0.85}{\milli\liter\per\minute} \\
			Temperature & \SI{25}{\celsius} 	\\
			Injection Volume 	& \SI{50}{\micro\liter} 	\\
			\bottomrule
		\end{tabularx}
	\end{table}

	\begin{table}[htbp]
		\caption[Standard aminocolumn method]{\textbf{Standard aminocolumn method}}
		\label{tab:method_nh2_standard}
		\centering
		\begin{tabularx}{\textwidth}{XX}
			\toprule
			\textbf{Parameter}	& \textbf{Value}	\\
			\midrule
			Column 		& Luna NH2 \SI{5}{\micro\meter} 250$\times$\SI{4.6}{\milli\meter} 	\\
			Solvents	& A: Water + 0.1~\% Formic acid 	\\
						& B: Acetonitrile + 0.1~\% Formic acid		\\
			Method 		& Isocratic 80~\% B for \SI{20}{\minute} 	\\
						& + 100~\% A for \SI{10}{\minute}   \\
			Flow 		& \SI{2}{\milli\liter\per\minute} \\
			Temperature & \SI{25}{\celsius} 	\\
			Injection Volume 	& \SI{50}{\micro\liter} 	\\
			\bottomrule
		\end{tabularx}
	\end{table}

	\begin{table}[htbp]
		\caption[Aminocolumn method adapted for MS coupling]{\textbf{Aminocolumn method adapted for MS coupling}}
		\label{tab:method_nh2_ms}
		\centering
		\begin{tabularx}{\textwidth}{XX}
			\toprule
			\textbf{Parameter}	& \textbf{Value}	\\
			\midrule
			Column 		& Luna NH2 \SI{5}{\micro\meter} 250$\times$\SI{4.6}{\milli\meter} 	\\
			Solvents	& A: Water + 0.1~\% Formic acid 	\\
						& B: Acetonitrile + 0.1~\% Formic acid		\\
			Method 		& Isocratic 80~\% B for 60 min. 	\\
			Flow 		& \SI{0.5}{\milli\liter\per\minute} \\
			Temperature & \SI{40}{\celsius} 	\\
			Injection Volume 	& \SI{50}{\micro\liter} 	\\
			\midrule
			Capillary Voltage 		& \SI{3500}{\volt} 	\\
			Injector Temperature	& \SI{350}{\celsius}\\
			Target mass 			& 400 m/z 			\\
			\bottomrule
		\end{tabularx}
	\end{table}

	\begin{table}[htbp]
		\caption[The standard ZIC-HILIC method]{\textbf{The standard ZIC-HILIC method}}
		\label{tab:method_hilic_standard}
		\centering
		\begin{tabularx}{\textwidth}{XX}
			\toprule
			\textbf{Parameter}	& \textbf{Value}	\\
			\midrule
			Column 		& ZIC-HILIC \SI{3.5}{\micro\meter} 150$\times$\SI{4.6}{\milli\meter} 	\\
			Solvents	& A: 	10~mM Ammonium acetate 	\\
						& B: 	Acetonitrile 			\\
			Method 		& Isocratic 80~\% B for 45 min. 	\\
			Flow 		& \SI{0.8}{\milli\liter\per\minute} \\
			Temperature & \SI{25}{\celsius} 	\\
			Injection Volume 	& \SI{50}{\micro\liter} 	\\
			\bottomrule
		\end{tabularx}
	\end{table}

	\begin{table}[htbp]
		\caption[ZIC-HILIC method adapted for MS coupling]{\textbf{ZIC-HILIC method adapted for MS coupling}}
		\label{tab:method_hilic_ms}
		\centering
		\begin{tabularx}{\textwidth}{XX}
			\toprule
			\textbf{Parameter}	& \textbf{Value}	\\
			\midrule
			Column 		& ZIC-HILIC \SI{3.5}{\micro\meter} 150$\times$\SI{4.6}{\milli\meter} 	\\
			Solvents	& A: 	10~mM Ammonium acetate 	\\
						& B: 	Acetonitrile 			\\
			Method 		& Isocratic 80~\% B for 60 min. 	\\
			Flow 		& \SI{0.5}{\milli\liter\per\minute} \\
			Temperature & \SI{40}{\celsius} 	\\
			Injection Volume 	& \SI{50}{\micro\liter} 	\\
			\midrule
			Capillary Voltage 		& \SI{3500}{\volt} 	\\
			Injector Temperature	& \SI{350}{\celsius}\\
			Target mass 			& 400 m/z 			\\
			\bottomrule
		\end{tabularx}
	\end{table}

	\begin{table}[htbp]
		\caption[Screening method for HPLC-MS]{\textbf{Screening method for HPLC-MS}}
		\label{tab:method_ms_1}
		\centering
		\begin{tabularx}{\textwidth}{XX}
			\toprule
			\textbf{Parameter}	& \textbf{Value}	\\
			\midrule
			Column 		& Nucleosil-100 \SI{5}{\micro\meter} 150$\times$\SI{3}{\milli\meter} 	\\
			Solvents	& A: Water + 0.1~\% Formic acid 	\\
						& B: Acetonitrile + 0.06~\% Formic acid		\\
			Method 		& Gradient 0 - 100 \% B for \SI{15}{\minute} 	\\
						& Plateau 100 \% B for \SI{2}{\minute} 	\\
			Flow 		& \SI{0.4}{\milli\liter\per\minute} \\
			Temperature & \SI{40}{\celsius} 	\\
			Injection Volume 	& \SI{2.5}{\micro\liter} 	\\
			\midrule
			Capillary Voltage 	& \SI{3500}{\volt} 	\\
			Injector Temperature& \SI{350}{\celsius}\\
			Target mass 		& 400 m/z 			\\
			\bottomrule
		\end{tabularx}
	\end{table}

	\begin{table}[htbp]
		\caption[Screening Method Polar-C18]{\textbf{Screening Method Polar-C18}}
		\label{tab:method_polarc18_screening}
		\centering
		\begin{tabularx}{\textwidth}{XX}
			\toprule
			\textbf{Parameter}	& \textbf{Value}	\\
			\midrule
			Column 		& Kinetex Polar-C18 \SI{2.6}{\micro\meter} 150$\times$\SI{4.6}{\milli\meter} 	\\
			Solvents	& A: Water + 0.1~\% Formic acid 	\\
						& B: Acetonitrile + 0.1~\% Formic acid		\\
			Method 		& Gradient 5 - 100 \% B for \SI{20}{\minute} 	\\
						& Plateau 100 \% B for \SI{6}{\minute} 	\\
			Flow 		& \SI{1.2}{\milli\liter\per\minute} \\
			Temperature & \SI{50}{\celsius} 	\\
			Injection Volume 	& \SI{50}{\micro\liter} 	\\
			\bottomrule
		\end{tabularx}
	\end{table}

	\begin{table}[htbp]
		\caption[Reverse Screening Method Polar-C18]{\textbf{Reverse Screening Method Polar-C18}}
		\label{tab:method_polarc18_revscreening}
		\centering
		\begin{tabularx}{\textwidth}{XX}
			\toprule
			\textbf{Parameter}	& \textbf{Value}	\\
			\midrule
			Column 		& Kinetex Polar-C18 \SI{2.6}{\micro\meter} 150$\times$\SI{4.6}{\milli\meter} 	\\
			Solvents	& A: Water + 0.1~\% Formic acid 	\\
						& B: Acetonitrile + 0.1~\% Formic acid		\\
			Method 		& Gradient 100 - 5 \% B for \SI{20}{\minute} 	\\
						& Plateau 100 \% B for \SI{6}{\minute} 	\\
			Flow 		& \SI{1.2}{\milli\liter\per\minute} \\
			Temperature & \SI{50}{\celsius} 	\\
			Injection Volume 	& \SI{50}{\micro\liter} 	\\
			\bottomrule
		\end{tabularx}
	\end{table}

% section hplc_methods (end)

\section{Chromatographic Data}
\label{sec:chromatographic_data}

\begin{figure}[htbp]
	\includegraphics[width=\textwidth]{app_nh2_chrom1}
	\caption[UV-chromatogram of reverse extract fractionation with the \luna column.]{\textbf{UV-chromatogram of reverse extract fractionation with the \luna column.}testi}
	\label{fig:app_nh2_chrom1}
\end{figure}

\section{Genomic Analysis} % (fold)
\label{sec:genomic_analysis}

\subsection{Phylogenetic Data}

Table including the reference genomes \ref{tab:ref_genomes}

\newlength{\asdf} %definiere neue länge

\setlength{\asdf}{\textwidth} %setze neue länge auf textbreite
\addtolength{\asdf}{-8\tabcolsep} %subtrahiere -8\cdot textbreite von asdf
% \begin{table}[htbp]
\begin{longtable}{p{0.5\asdf}p{0.5\asdf}}
\caption{Reference genomes for the construction of the phylogenetic tree}
\label{tab:ref_genomes} \\
% \centering
% \begin{tabularx}{ll}
	\toprule
	\textbf{Assembly ID} & \textbf{Species name}		\\
	\midrule
	GCF\_000010605 	&	\emph{S. griseus} subsp. griseus NBRC 13350	\\
	GCF\_000156455 	&	\emph{S. roseosporus} NRRL 15998	\\
	GCF\_000156695 	&	\emph{S. roseosporus} NRRL 11379	\\
	GCF\_000177175 	&	\emph{Streptomyces} sp. ACT-1	\\
	GCF\_000239075 	&	\emph{Streptomyces} sp. W007	\\
	GCF\_000261345 	&	\emph{S. globisporus} C-1027	\\
	GCF\_000373305 	&	\emph{Streptomyces} sp. CcalMP-8W	\\
	GCF\_000373405 	&	\emph{Streptomyces} sp. ScaeMP-e10	\\
	GCF\_000377965 	&	\emph{Streptomyces} sp. CNB091	\\
	GCF\_000385945 	&	\emph{S. fulvissimus} DSM 40593	\\
	GCF\_000498935 	&	\emph{Streptomyces} sp. HCCB10043	\\
	GCF\_000647875 	&	\emph{Streptomyces} sp. SolWspMP-sol2th	\\
	GCF\_000702365 	&	\emph{Streptomyces} sp. CNS654	\\
	GCF\_000716025 	&	\emph{S. baarnensis}	\\
	GCF\_000716935 	&	\emph{S. alboviridis}	\\
	GCF\_000716945 	&	\emph{S. albus subsp. albus}	\\
	GCF\_000717015 	&	\emph{S. brasiliensis}	\\
	GCF\_000717105 	&	\emph{S. anulatus}	\\
	GCF\_000717215 	&	\emph{S. vinaceus}	\\
	GCF\_000717645 	&	\emph{S. californicus}	\\
	GCF\_000717665 	&	\emph{S. floridae}	\\
	GCF\_000717795 	&	\emph{S. griseus} subsp. griseus	\\
	GCF\_000717965 	&	\emph{S. purpeochromogenes}	\\
	GCF\_000718135 	&	\emph{S. cyaneofuscatus}	\\
	GCF\_000718205 	&	\emph{S. griseus} subsp. rhodochrous	\\
	GCF\_000718235 	&	\emph{S. griseus }subsp. rhodochrous	\\
	GCF\_000718245 	&	\emph{S. californicus}	\\
	GCF\_000718455 	&	\emph{S}\emph{. globisporus} subsp. globisporus	\\
	GCF\_000718615 	&	\emph{S. purpeochromogenes}	\\
	GCF\_000718695 	&	\emph{S. puniceus}	\\
	GCF\_000718915 	&	\emph{Streptomyces} sp. NRRL WC-3540	\\
	GCF\_000719035 	&	\emph{S. griseus} subsp. rhodochrous	\\
	GCF\_000719195 	&	\emph{S. puniceus}	\\
	GCF\_000719355 	&	\emph{S. griseus} subsp. griseus	\\
	GCF\_000719585 	&	\emph{Streptomyces} sp. NRRL F-5681	\\
	GCF\_000719655 	&	\emph{Streptomyces} sp. NRRL B-1381	\\
	GCF\_000719925 	&	\emph{Streptomyces} sp. NRRL F-2202	\\
	GCF\_000720055 	&	\emph{Streptomyces} sp. NRRL F-3218	\\
	GCF\_000720065 	&	\emph{Streptomyces} sp. NRRL F-3273	\\
	GCF\_000720915 	&	\emph{S. albus} subsp. albus	\\
	GCF\_000721175 	&	\emph{S. anulatus}	\\
	GCF\_000721205 	&	\emph{S. griseus} subsp. rhodochrous	\\\emph{}
	GCF\_000721415 	&	\emph{Streptomyces} sp. NRRL F-5702	\\
	GCF\_000721575 	&	\emph{S. griseus} subsp. griseus	\\
	GCF\_000721685 	&	\emph{S. mediolani}	\\
	GCF\_000725705 	&	\emph{Streptomyces} sp. NRRL S-623	\\
	GCF\_000743295 	&	\emph{Streptomyces} sp. JS01	\\
	GCF\_000932225 	&	\emph{S. griseus}	\\
	GCF\_000935135 	&	\emph{Streptomyces} sp. MNU77	\\
	GCF\_001189025 	&	\emph{S. europaeiscabiei}	\\
	GCF\_001270675 	&	\emph{S. griseus} subsp. rhodochrous	\\
	GCF\_001278095 	&	\emph{Streptomyces} sp. CFMR 7	\\
	GCF\_001279425 	&	\emph{Streptomyces} sp. NRRL F-2295	\\
	GCF\_001279735 	&	\emph{Streptomyces} sp. MMG1522	\\
	GCF\_001418625 	&	\emph{S. luridiscabiei}	\\
	GCF\_001426325 	&	\emph{Streptomyces} sp. Root1295	\\
	GCF\_001427565 	&	\emph{Streptomyces} sp. Root63	\\
	GCF\_001434355 	&	\emph{S. anulatus}	\\
	GCF\_001687325 	&	\emph{Streptomyces} sp. PTY087I2	\\
	GCF\_001723115 	&	\emph{S. griseus}	\\
	GCF\_001723125 	&	\emph{S. griseus} subsp. griseus	\\
	GCF\_001746285 	&	\emph{Streptomyces} sp. EN16	\\
	GCF\_001746305 	&	\emph{Streptomyces} sp. EN23	\\
	GCF\_001746315 	&	\emph{Streptomyces} sp. EN27	\\
	GCF\_001751305 	&	\emph{S. nanshensis}	\\
	GCF\_001895105 	&	\emph{Streptomyces} sp. NBRC 110465	\\
	GCF\_001905405 	&	\emph{Streptomyces} sp. TSRI0445	\\
	GCF\_001905485 	&	\emph{Streptomyces} sp. TSRI0261	\\
	GCF\_001905525 	&	\emph{Streptomyces} sp. TSRI0395	\\
	GCF\_001905575 	&	\emph{Streptomyces} sp. CB00072	\\
	GCF\_001905595 	&	\emph{Streptomyces} sp. CB00316	\\
	GCF\_001905645 	&	\emph{Streptomyces} sp. CB02115	\\
	GCF\_001905785 	&	\emph{Streptomyces} sp. CB02130	\\
	GCF\_001905905 	&	\emph{Streptomyces} sp. CB02366	\\
	GCF\_001931635 	&	\emph{Streptomyces} sp. Tue 6075	\\
	GCF\_001983595 	&	\emph{Streptomyces} sp. IB2014 011-1	\\
	GCF\_002082175 	&	\emph{S. violaceoruber}	\\
	GCF\_002082585 	&	\emph{Kitasatospora albolonga}	\\
	GCF\_002094995 	&	\emph{Streptomyces} sp. S8	\\
	GCF\_002154345 	&	\emph{S. fimicarius}	\\
	GCF\_002154385 	&	\emph{S. albovinaceus}	\\
	GCF\_002188345 	&	\emph{Streptomyces} sp. CS057	\\
	GCF\_002217715 	&	\emph{Streptomyces} sp. SS07	\\
	GCF\_002242735 	&	\emph{Streptomyces} sp. 2R	\\
	GCF\_900090075 	&	\emph{Streptomyces} sp. MnatMP-M77	\\
	GCF\_900090085 	&	\emph{Streptomyces} sp. Ncost-T6T-1	\\
	GCF\_900090135 	&	\emph{Streptomyces} sp. LaPpAH-199	\\
	GCF\_900091775 	&	\emph{Streptomyces} sp. ScaeMP-e83	\\
	GCF\_900091865 	&	\emph{Streptomyces} sp. DvalAA-19	\\
	GCF\_900091905 	&	\emph{Streptomyces} sp. OspMP-M43	\\
	GCF\_900092005 	&	\emph{Streptomyces} sp. Cmuel-A718b	\\
	GCF\_900105705 	&	\emph{S. griseus}	\\
	GCF\_900187925 	&	\emph{Streptomyces} sp. PgraA7	\\
	\bottomrule
% \end{tabularx}
% \end{table}
\end{longtable}
	Table including the identified single-copy genes

	\begin{longtable}{p{0.25\asdf}p{0.75\asdf}}
		\caption{Single copy genomes for tree construction}
		\label{tab:single_copy_genes} \\
		% \centering
		% \begin{tabularx}{\textwidth}{>{\hsize=.5\hsize}X>{\hsize=1.5\hsize}X}
			\toprule
			\textbf{TIGR ID} 			& \textbf{Description}		\\
			\midrule
			00008 	& translation initiation factor IF-1	\\
			00033	& chorismate synthase 					\\
			00060	& ribosomal protein uL18 				\\
			00062	& ribosomal protein bL27 				\\
			00118	& acetolactate synthase, large subunit, biosynthetic type	\\
			00151	& 2-C-methyl-D-erythritol 2,4-cyclodiphosphate synthase		\\
			00171	& 3-isopropylmalate dehydratase, small subunit				\\
			00302	& phosphoribosylformylglycinamidine synthase, purS protein	\\
			00355	& phosphoribosylaminoimidazolecarboxamide formyltransferase/IMP cyclohydrolase	\\
			00382	& ATP-dependent Clp protease, ATP-binding subunit ClpX		\\
			00431	& tRNA pseudouridine(55) synthetase		\\
			00484	& translation elongation factor G		\\
			00615	& recombination protein RecR			\\
			00631	& exinuclease ABC subunit B 			\\
			00708	& cob(I)yrinic acid a,c-diamide adenosyltransferase			\\
			00962	& ATP synthase F1, alpha subunit 		\\
			00981	& ribosomal protein uS12				\\
			01009	& ribosomal protein uS3					\\
			01021	& ribosomal protein uS5					\\
			01022	& ribosomal protein bL36				\\
			01024	& ribosomal protein bL19				\\
			01029	& ribosomal protein uS7					\\
			01030	& ribosomal protein bL34				\\
			01032	& ribosomal protein bL20				\\
			01039	& ATP synthase F1, beta subunit			\\
			01044	& ribosomal protein uL22				\\
			01049	& ribosomal protein uS10				\\
			01050	& ribosomal protein uS19				\\
			01067	& ribosomal protein uL14				\\
			01083	& endonuclease III						\\
			01134	& amidophophoribosyltransferase			\\
			01162	& phosphoribosylaminoimidazole carboxylase, catalytic subunit	\\
			01164	& ribosomal protein uL16 				\\
			01169	& ribosomal protein uL1					\\
			01171	& ribosomal protein uL2					\\
			01393	& elongation factor 4					\\
			01980	& FeS assembly protein SufB				\\
			02013	& DNA-directed RNA polymerase, beta subunit		\\
			02027	& DNA-directed RNA polymerase, alpha subunit	\\
			02156	& phenylacetate-CoA oxygenase, PaaG subunit		\\
			02157	& phenylacetate-CoA oxygenase, PaaH subunit		\\
			02952	& RNA polymerase sigma-70 factor		\\
			03188	& phosphoribosyl-ATP diphosphatase		\\
			03450	& inositol 1-phosphate synthase			\\
			03631	& ribosomal protein uS13				\\
			03632	& ribosomal protein uS11				\\
			03687	& ubiquitin-like protein Pup			\\
			03699	& dehypoxanthine futalosine cyclase		\\
			03800	& pyridoxal 5'-phosphate synthase, glutaminase subunit Pdx2	\\
			\bottomrule
		% \end{tabularx}
	\end{longtable}

%	\begin{table}[htbp]
%	\begin{longtable}{p{.25\textwidth}p{.75\textwidth}}
%		\caption{Single copy genomes for tree construction}
%%		\label{tab:single_copy_genes}
%%		\centering
%		\toprule
%		\textbf{TIGR ID} 			& \textbf{Description}		\\
%		\midrule
%		00008 	& translation initiation factor IF-1	\\
%		00033	& chorismate synthase 					\\
%		00060	& ribosomal protein uL18 				\\
%		00062	& ribosomal protein bL27 				\\
%		00118	& acetolactate synthase, large subunit, biosynthetic type	\\
%		00151	& 2-C-methyl-D-erythritol 2,4-cyclodiphosphate synthase		\\
%		00171	& 3-isopropylmalate dehydratase, small subunit				\\
%		00302	& phosphoribosylformylglycinamidine synthase, purS protein	\\
%		00355	& phosphoribosylaminoimidazolecarboxamide formyltransferase/IMP cyclohydrolase	\\
%		00382	& ATP-dependent Clp protease, ATP-binding subunit ClpX		\\
%		00431	& tRNA pseudouridine(55) synthetase		\\
%		00484	& translation elongation factor G		\\
%		00615	& recombination protein RecR			\\
%		00631	& exinuclease ABC subunit B 			\\
%		00708	& cob(I)yrinic acid a,c-diamide adenosyltransferase			\\
%		00962	& ATP synthase F1, alpha subunit 		\\
%		00981	& ribosomal protein uS12				\\
%		01009	& ribosomal protein uS3					\\
%		01021	& ribosomal protein uS5					\\
%		01022	& ribosomal protein bL36				\\
%		01024	& ribosomal protein bL19				\\
%		01029	& ribosomal protein uS7					\\
%		01030	& ribosomal protein bL34				\\
%		01032	& ribosomal protein bL20				\\
%		01039	& ATP synthase F1, beta subunit			\\
%		01044	& ribosomal protein uL22				\\
%		01049	& ribosomal protein uS10				\\
%		01050	& ribosomal protein uS19				\\
%		01067	& ribosomal protein uL14				\\
%		01083	& endonuclease III						\\
%		01134	& amidophophoribosyltransferase			\\
%		01162	& phosphoribosylaminoimidazole carboxylase, catalytic subunit	\\
%		01164	& ribosomal protein uL16 				\\
%		01169	& ribosomal protein uL1					\\
%		01171	& ribosomal protein uL2					\\
%		01393	& elongation factor 4					\\
%		01980	& FeS assembly protein SufB				\\
%		02013	& DNA-directed RNA polymerase, beta subunit		\\
%		02027	& DNA-directed RNA polymerase, alpha subunit	\\
%		02156	& phenylacetate-CoA oxygenase, PaaG subunit		\\
%		02157	& phenylacetate-CoA oxygenase, PaaH subunit		\\
%		02952	& RNA polymerase sigma-70 factor		\\
%		03188	& phosphoribosyl-ATP diphosphatase		\\
%		03450	& inositol 1-phosphate synthase			\\
%		03631	& ribosomal protein uS13				\\
%		03632	& ribosomal protein uS11				\\
%		03687	& ubiquitin-like protein Pup			\\
%		03699	& dehypoxanthine futalosine cyclase		\\
%		03800	& pyridoxal 5'-phosphate synthase, glutaminase subunit Pdx2	\\
%		\bottomrule
%	\end{longtable}
%\end{table}

    \begin{figure}[htpb]
        \centering
        \begin{subfigure}[b]{\textwidth}
            \includegraphics[width=1\linewidth]{contig4_cluster_search}
            \caption{Cluster search results for the identified cluster.}
            \label{fig:sub1}
        \end{subfigure}

        \begin{subfigure}[b]{\textwidth}
            \includegraphics[width=1\linewidth]{contig4_subcluster_search}
            \caption{Subcluster search results for the identified cluster.}
            \label{fig:sub2}
        \end{subfigure}

        \caption[Cluster and subcluster search results for the cluster located on contig 4.]{\textbf{Cluster and subcluster search results for the cluster located on contig 4.} The 160~kb contig was submitted to AntiSMASH with the ClusterFinder option. Only the search results with the highest similarities are shown.}
        \label{fig:cluster_search}
    \end{figure}

    % section genomic_analysis (end)



%%% Local Variables:
%%% mode: latex
%%% TeX-master: "../main"
%%% End:
